frontend/apps/reader/skimtowidget.lua

```diff
@@ -399,6 +421,20 @@ function SkimToWidget:onAnyKeyPressed()
     return true
 end
 
+function SkimToWidget:onSelectByKeyPress()
+    local item = self:getFocusItem()
+    item.callback()
+end
+
+function SkimToWidget:onFirstRowKeyPress(args)
+        local perc = args
```

Comment:
nit

Reply:
Well, double nit. I'm not really sure why you're not using `onFirstRowKeyPress(percent)` the line above, but I was talking about the indentation.

Reply:
Thanks for checking! Do i need to replace perc with args?